docs/operations-guide/start.md

```diff
@@ -8,6 +8,7 @@
 *  [Migrating from using the H2 database to MySQL or Postgres](#migrating-from-using-the-h2-database-to-mysql-or-postgres)
 *  [Running database migrations manually](#running-metabase-database-migrations-manually)
 *  [Backing up Metabase Application Data](#backing-up-metabase-application-data)
+*  [Storing database connection details more securely](#storing-database-connection-details-more-securely)
```

Comment:
think we should make it clear we're taking about encryption
"Encrypting your database connection details at rest" or something similar might be clearer.